enterprise/frontend/src/metabase-enterprise/embedding/components/EmbeddingAppOriginDescription/EmbeddingAppOriginDescription.tsx

```diff
@@ -1,31 +1,18 @@
 import { jt, t } from "ttag";
 import ExternalLink from "metabase/core/components/ExternalLink";
-import MetabaseSettings from "metabase/lib/settings";
 
 export const EmbeddingAppOriginDescription = () => {
   return (
-    <div>
-      {jt`With this Pro/Enterprise feature you can embed the full Metabase app. Enable your users to drill-through to charts, browse collections, and use the graphical query builder. ${(
+    <div className="my4">
+      <strong className="block text-dark mb1">{t`Authorized origins`}</strong>
```

Comment:
Why do we have to manually code the settings title and not use `display_name` in the setting instead? Using `strong` makes the settings headers look different than other settings in admin settings which seems undesirable.

#### using `display_name` + some spacing adjustments
![image](https://github.com/metabase/metabase/assets/1937582/c00fc4d9-dc83-4ca5-955c-df9991e39bb0)

#### This PR
![image](https://github.com/metabase/metabase/assets/1937582/61cd8f57-9f28-4cd1-ab7b-3626a80cfcf3)

Reply:
Fixed in https://github.com/metabase/metabase/pull/37043/commits/41bee969d22ec1a2bde8d26173f8e9dfe6939628